lib/modules/manager/gradle-wrapper/artifacts.ts

```diff
@@ -194,6 +194,48 @@ export async function updateArtifacts({
       );
     }
 
+    let writeLocksCmd = await prepareGradleCommand(gradlewFile);
```

Comment:
This function should only need to call `prepareGradleCommand()` once and after that the value should be reused.

Reply:
If we move forward with leaving out the `--write-locks` portion of the PR, I can probably just reuse `cmd` as is since including the `gradle-distribution-url` for the second execution would work as well.